api/src/main/java/info/rmapproject/api/service/EventApiService.java

```diff
@@ -172,7 +172,7 @@ public Response apiGetRMapEvent(@Context HttpHeaders headers, @PathParam("eventU
     @Produces({"application/json;charset=UTF-8;","text/plain;charset=UTF-8;"})
     public Response apiGetRMapEventDiSCOs(@Context HttpHeaders headers, @PathParam("eventUri") String eventUri) throws RMapApiException {
     	NonRdfType outputType = HttpTypeMediator.getNonRdfResponseType(headers);
-    	Response relatedDiscos = getEventResponseManager().getRMapEventRelatedObjs(eventUri, RMapObjectType.EVENT, outputType);
+    	Response relatedDiscos = getEventResponseManager().getRMapEventRelatedObjs(eventUri, RMapObjectType.DISCO, outputType);
```

Comment:
I don't know enough about the codebase to know if this is correct or not so just trusting @karenhanson's judgement on this one.

Reply:
That's fine - the links under the testing section show the issue - it returns a list of DiSCOs, but the JSON was labeling them as Events. It's a very minor change.

Reply:
👍